dct_backend/food_review.py

```diff
@@ -0,0 +1,65 @@
+from food_item import FoodItem
+
+
+class FoodReview ():
```

Comment:
I see how you use this can be used to manage reviews locally, but we don't care for local instances of reviews. We want to take data and plop it into the database OR find data from the database and plop it to the user. How will this class help you do that?